libs/coin-modules/coin-tron/src/api/index.integ.test.ts

```diff
@@ -0,0 +1,63 @@
+import { type Api } from "@ledgerhq/coin-framework/api/index";
+import { createApi } from ".";
+/**
+ * https://api.shasta.trongrid.io
+ */
+describe("Tron Api", () => {
+  let module: Api;
+  // const address = "tz1heMGVHQnx7ALDcDKqez8fan64Eyicw4DJ";
+
+  beforeAll(() => {
+    module = createApi({
+      explorer: {
+        url: "https://api.shasta.trongrid.io",
+      },
+    });
+  });
+
+  describe("broadcast", () => {
+    it.skip("returns tx id", async () => {
```

Comment:
Not sure if it's possible to test without local node

Unit test should be enough